### PR TITLE
Fixes radiation wave runtime.

### PR DESCRIPTION
**Pull request by nemvar**
¯\_(ツ)_/¯ get_step sometimes returns null and I'm not sure why. This should fix the runtime though.

### CODE REVIEW

**Comment by optimumtact** [comment]
this doesn't fix anything, it suppresses the runtime which is silly

**Comment by mrhugo13** [comment]
> 
> 
> this doesn't fix anything, it suppresses the runtime which is silly

Don't you get it Oranges? We must supress the anti-gamer-karen-runtimes.

**Comment by Dennok** [comment]
It runtime if try to out of bounds of map?
So what fix for this can be?

**Comment by nemvar** [comment]
Does it really just surpress the runtime if it's consistent with how rad waves work otherwise?

**Comment by ninjanomnom** [comment]
This isn't suppressing anything bad, radiation waves check that they arent exiting the map in the primary direction of movement but I forgot to add a check for when the width goes outside map bounds

**Comment by optimumtact** [comment]
yeah but shouldn't you check instead, rather than supressing it all the way here

File: code/datums/radiation_wave.dm [hunk 1]
```diff
@@ -68,7 +68,8 @@
 		place = cmaster_turf
 		for(var/i in 1 to distance)
 			place = get_step(place, dir)
-			atoms += get_rad_contents(place)
+			if(place)
+				atoms += get_rad_contents(place)
```
Comment (ninjanomnom) [inline code/datums/radiation_wave.dm]: If you hit a null location you don't need to continue trying this direction, you can skip to the next one. Break out of this loop instead
```
if(!place)
  break
```

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by ninjanomnom** [comment]
This is where the check should happen, it's just doing a slightly wrong thing because of the check at the moment. A break there makes it stop trying to get locations off the map edge

**Comment by kriskog** [comment]
\\\¯\\\_(ツ) you dropped this

**Comment by nemvar** [comment]
¯ \_(ツ)_/¯

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**